### ISSUE TITLE
`FileNotFoundException` exception not documented

### ISSUE THREAD

**Issue by vgromfeld** [MicrosoftDocs/winrt-api#1892]
The API is throwing a `FileNotFoundException` if the item registered in the access list has been deleted. The documentation should mention this (and the other exceptions if any).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 91ec24e6-4948-f1c9-9d54-d142f9f7296f
* Version Independent ID: ec401f73-0a6a-01b5-16cb-634191f0f683
* Content: [IStorageItemAccessList.GetItemAsync Method (Windows.Storage.AccessCache) - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.storage.accesscache.istorageitemaccesslist.getitemasync?view=winrt-19041)
* Content Source: [winrt-api-build/xml/Windows.Storage.AccessCache/IStorageItemAccessList.xml](https://github.com/MicrosoftDocs/winrt-api/blob/docs/winrt-api-build/xml/Windows.Storage.AccessCache/IStorageItemAccessList.xml)
* Product: **uwp**
* Technology: **winrt-reference**
* GitHub Login: @lastnameholiu
* Microsoft Alias: **alholiu**

**Comment by mikerapid** [comment]
Hello, thanks for your feedback on the documentation. WinRT APIs are designed such that exceptions are intended to provide feedback during app development and should not be caught at runtime. This is one reason our convention is not to document exceptions for WinRT APIs. The exception type and message should be sufficient to let you know the necessary changes in your code.

**Comment by vgromfeld** [comment]
I agree for general exception like `ArgumentNullException` or things like that. but for this method. It is better to have documented than discovering it through a crash of the application.

**Comment by GrantMeStrength** [comment]
Closing this issue due to age (filed before 2024). If this is still relevant, please reopen or file a new issue with updated details. Thank you for your contribution!
